packages/components/src/schema/props/badge-text.ts

```diff
@@ -0,0 +1,19 @@
+import type { Generic } from 'adopted-style-sheets';
+import { watchString } from '../utils';
+import type { AccessKeyPropType } from './access-key';
+import type { ShortKeyPropType } from './short-key';
+
+/* types */
+export type BadgeTextPropType = string;
+
+/**
+ * Defines the elements badge text.
+ */
+export type PropBadgeText = {
+	badgeText: AccessKeyPropType | ShortKeyPropType;
```

Comment:
```suggestion
	badgeText: BadgeTextPropType;
```

Hier sollte der oben definierte PropType verwendet werden. In L7 können wir dafür die Union `AccessKeyPropType | ShortKeyPropType` statt dem hardcoded string verwenden.